README.md

```diff
@@ -1 +1,30 @@
-This is the Intercept plugin template project.
+# what
+
+use MariaDB / MySQL in Arma
+
+# how
+
+define MariaDB or MySQL connection parameters by editing `config.yaml` in the mod directory:
+
+```yaml
+accounts: 
+  foo:
+    ip: 127.0.0.1
+    username: us3r
+    password: s3cr3t
+    database: foo 
+statements:
+  foostmt: "select 1"
+```
+
+and execute/retrieve results like this:
+
+```sqf
+private _connection = dbCreateConnection "foo"; 
+private _query = dbPrepareQuery "foostmt"; 
+private _result = _connection dbExecute _query; 
+private _resultArray = dbResultToArray _result;
+systemChat format ["xxx %1", _resultArray];
```

Comment:
up until this point, there are no errors - but formatting the `_resultArray` results in "array" , which is a bit weird  - I'd expect "[1]"  🤔

Reply:
"array" looks like it's nil.
Which means _result is nil? wat..

Reply:
So from top to bottom.
`createConnectionConfig` will return nil if the account doesn't exist in config...
That might be it, though your overall code is correct. I need to add a warning to that.

`dbPrepareQuery` can't return nil.
`dbExecute` can't return nil, but crashes game if query fails in unscheduled (already fixed on master)
`dbResultToArray` returns empty array if result wasn't correct.

So only cause I can see is createConnection not finding the account in config.

Reply:
yes, please warn the shit out of us, or throw errors or … something. 
`nil` is the Value Of Satan , I never want to see it :D

Reply:
I'll have a look at the config.yaml I used. :thinking: